Documentation/subcommands/prepare.md

```diff
@@ -52,6 +52,8 @@ c9fad0e6-8236-4fc2-ad17-55d0a4c7d742
 | `--stage1-from-dir` |  `` | A stage1 image file inside the default stage1 images directory | Image to use as stage1 |
 | `--store-only` |  `false` | `true` or `false` | Use only available images in the store (do not discover or download from remote URLs). See [image fetching behavior](../image-fetching-behavior.md) |
 | `--volume` |  `` | Volume syntax (`NAME,kind=KIND,source=PATH,readOnly=BOOL`). See [Mount Volumes into a Pod](run.md#mount-volumes-into-a-pod) | Volumes to make available in the pod |
+| `--user` | none | username or UID | user override for the preceding image (example: '--user=user') |
```

Comment:
Capitalize first letters in "username" and "user". Like it is done in other table rows.